src/features/dialog-manager/dialogs/character-customisation/index.js

```diff
@@ -0,0 +1,107 @@
+
+import ColourPicker from '../../../../components/colour-picker';
+
+import './styles.scss';
+
+const CharacterCustomisation = ({
+    dialog,
+    finishCustomisation,
+    setHairColour,
+    setEyeColour,
+    setSkinColour,
+    setArmourColour,
+    setClothesColour,
+}) => {
+    const {
+        hairColour,
+        eyeColour,
+        skinColour,
+        armourColour,
+        clothesColour,
+    } = dialog.appearance;
+
+    return (
+        <>
+            <Dialog onKeyPress={finishCustomisation}>
+                <div className="flex-column character-customisation__container">
+                    <div className="flex-column character-customisation__title">
+                        Customise Character
+                    </div>
+                    <Character
+                        hairColour={hairColour}
+                        eyeColour={eyeColour}
+                        skinColour={skinColour}
+                        armourColour={armourColour}
+                        clothesColour={clothesColour}
+                    />
+                    <div className="flex-column character-customisation__options-container">
+                        <div className="flex-row character-customisation__option">
+                            <p className="character-customisation__label">
+                                Hair
+                            </p>
+                            <div className="character-customisation__picker">
+                                <ColourPicker onChange={setHairColour} />
+                            </div>
+                        </div>
+                        <div className="flex-row character-customisation__option">
+                            <p className="flex-column character-customisation__label">
+                                Skin
+                            </p>
+                            <div className="character-customisation__picker">
+                                <ColourPicker onChange={setSkinColour} />
+                            </div>
+                        </div>
+                        <div className="flex-row character-customisation__option">
+                            <p className="flex-column character-customisation__label">
+                                Armour
+                            </p>
+                            <div className="character-customisation__picker">
+                                <ColourPicker onChange={setArmourColour} />
+                            </div>
+                        </div>
+                        <div className="flex-row character-customisation__option">
+                            <p className="flex-column character-customisation__label">
+                                Clothes
+                            </p>
+                            <div className="character-customisation__picker">
+                                <ColourPicker onChange={setClothesColour} />
+                            </div>
```

Comment:
Could we modify this similar to how we did the Select Buttons and Ability Scores?